packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -113,7 +127,12 @@ export class JobService {
       .div(100);
     const totalAmount = BigNumber.from(fundAmountInWei).add(totalFee);
 
-    if (userBalance.lte(totalAmount)) {
+    const fixedAmount = FixedNumber.from(
+      ethers.utils.formatUnits(totalAmount, 18),
```

Comment:
lets try to use the unit name for this kind of conversions to make the code easier to understand:

```suggestion
      ethers.utils.formatUnits(totalAmount, 'eth'),
```